hubble/pkg/printer/printer.go

```diff
@@ -280,12 +280,17 @@ func formatPolicyNames(policies []*flowpb.Policy) string {
 }
 
 func (p Printer) getSummary(f *flowpb.Flow) string {
-	auth := p.getAuth(f)
-	if auth == "" {
-		return f.GetSummary()
+	b := strings.Builder{}
+	if f.IpTraceId != nil && f.IpTraceId.TraceId > 0 {
+		b.WriteString(fmt.Sprintf("IP Trace ID: %d; ", f.IpTraceId.TraceId))
 	}
+	b.WriteString(f.GetSummary())
 
-	return fmt.Sprintf("%s; Auth: %s", f.GetSummary(), auth)
+	if auth := p.getAuth(f); auth != "" {
+		b.WriteString("; " + auth)
+	}
```

Comment:
I think this should be 
```suggestion
	if auth := p.getAuth(f); auth != "" {
		b.WriteString("; Auth:" + auth)
	}
```

Reply:
Fixed.